builder/builder.go

```diff
@@ -158,7 +189,7 @@ func (b *Builder) onSealedBlock(block *types.Block, blockValue *big.Int, ordersC
 		}
 	}
 
-	log.Info("submitted block", "slot", attrs.Slot, "value", blockValue.String(), "parent", block.ParentHash, "hash", block.Hash(), "#commitedBundles", len(commitedBundles))
+	log.Info("submitted block", "slot", attrs.Slot, "value", blockValue.String(), "parent", block.ParentHash, "hash", block.Hash(), "#commitedBundles", len(commitedBundles), "blockTimestamp", uint64(attrs.Timestamp))
```

Comment:
Is timestamp necessary if we already have the slot?
Same in a couple different places

Reply:
> Is timestamp necessary if we already have the slot?

Good question - out of an abundance of caution I added it separate to sanity check that the timestamp indeed match the slot. I don't know if we're checking elsewhere that timestamp always matches slot. This was mostly useful for me debugging so open to removing it

Reply:
i've removed timestamp since it was more for analysis of a separate issue unrelated to this PR - so going to have it on separate branch, removed it here